src/storage/system_catalog.c

```diff
@@ -1624,13 +1624,13 @@ catalog_drop_representation_helper (THREAD_ENTRY * thread_p, PAGE_PTR page_p, VP
 
   spage_get_record (thread_p, page_p, CATALOG_HEADER_SLOT, &record, COPY);
 
-  log_append_undo_recdes2 (thread_p, RVCT_UPDATE, &catalog_Id.vfid, page_p, CATALOG_HEADER_SLOT, &record);
-
   overflow_vpid.pageid = CATALOG_GET_PGHEADER_OVFL_PGID_PAGEID (record.data);
   overflow_vpid.volid = CATALOG_GET_PGHEADER_OVFL_PGID_VOLID (record.data);
 
   if (overflow_vpid.pageid != NULL_PAGEID)
     {
+      log_append_undo_recdes2 (thread_p, RVCT_UPDATE, &catalog_Id.vfid, page_p, CATALOG_HEADER_SLOT, &record);
```

Comment:
If the undo is appended on the same condition with the redo of it, why don't we use `log_append_undoredo_recdes2`?

Reply:
Yes, you are right. 

I also wanted to use the `log_append_undoredo_recdes2` function, but I followed the above method because undo and redo were left separately as above in all places logging using RVCT_UPDATE. 

After analysis, I think it is better to use `log_append_undoredo_recdes2` as you mentioned.